src/frontend/src/catalog.rs

```diff
@@ -283,11 +283,26 @@ impl CatalogManager for FrontendCatalogManager {
             .await
     }
 
-    fn as_any(&self) -> &dyn Any {
+    fn as_any(&self) -> &(dyn Any + Send + Sync) {
         self
     }
 }
 
+#[async_trait::async_trait]
+impl CachedCatalogManager for FrontendCatalogManager {
+    async fn invalidate_table_cache(&self, catalog: &str, schema: &str, table: &str) {
+        let tg_key = TableGlobalKey {
+            catalog_name: catalog.into(),
+            schema_name: schema.into(),
+            table_name: table.into(),
+        };
+
+        let tg_key: Vec<u8> = tg_key.to_string().into();
```

Comment:
We can avoid a clone here.
```suggestion
        let tg_key = TableGlobalKey {
            catalog_name: catalog.into(),
            schema_name: schema.into(),
            table_name: table.into(),
        }
        .to_string();

        let tg_key = tg_key.as_bytes();
```

Reply:
Thanks for your advice which is more elegant. but to clarify, `into()` here should not cause extra memory copy. 😃